db/migrate/20211227133345_populate_feedback_questions_migration.rb

```diff
@@ -0,0 +1,5 @@
+class PopulateFeedbackQuestionsMigration < ActiveRecord::Migration[6.0]
+  def change
+    Rake::Task['populate_feeback_data:process'].invoke
+  end
+end
```

Comment:
@mohsinmalik568 I don't think we need a migration to run a rack task. It's unnecessary, for running this rack task all we need to do is just to execute `Rake::Task['populate_feeback_data:process'].invoke` or even better  `rails populate_feeback_data: process `

Reply:
OK removed the migration please run rails populate_feeback_data: process when deploy production